### DISCUSSION TITLE
support external game translation

### DISCUSSION THREAD

**Discussion by kiritoash2021** [skyline-emu/skyline#643] [Ideas]
Hello, I would like you to put support for placing external game translation because there are many games that don't have translation into several languages

**Comment by QuackingCanary** [comment]
It's already planned, that's just a matter of time!

**Reply by kiritoash2021** [reply]
Good thing because I'm Brazilian and there aren't many titles that Nintendo makes Portuguese localization

**Comment by PixelyIon** [comment]
@Bibiloux98 is incorrect as that's about translations for Skyline itself. While the question is asking for additional translations of the games themselves.
@kiritoash2021 We cannot add translations to the games themselves aside from supporting mods. Games don't store their strings in a uniform format so unless someone releases a mod for a game it's not something we can add support for doing.

**Reply by kiritoash2021** [reply]
Ah sorry what I meant and external translation mod support as well as yuzu He does etc.

**Reply by PixelyIon** [reply]
We will eventually add support for mods so that should indeed work.

**Comment by ShadowBlakc** [comment]
Váyanse a la mierda
